guides/source/security.md

```diff
@@ -1173,31 +1173,31 @@ It is beyond the scope of this guide to inform you on how to secure your applica
 
 ### Custom Credentials
 
-Rails stores secrets in `config/credentials.yml.enc`, which is encrypted and hence cannot be edited directly. Rails uses `config/master.key` or alternatively looks for environment variable `ENV["RAILS_MASTER_KEY"]` to encrypt the credentials file. The credentials file can be stored in version control, as long as master key is kept safe.
+Rails stores secrets in `config/credentials.yml.enc`, which is encrypted and hence cannot be edited directly. Rails uses `config/master.key` or alternatively looks for environment variable `ENV["RAILS_MASTER_KEY"]` to encrypt the credentials file. Because the credentials file is encrypted, it can be stored in version control, as long as the master key is kept safe.
 
-To add new secret to credentials, first run `bin/rails secret` to get a new secret. Then run `bin/rails credentials:edit` to edit credentials, and add the secret. Running `credentials:edit` creates new credentials file and master key, if they did not already exist.
+By default, the credentials file contains the application's
+`secret_key_base`. It can also be used to store other secrets such as access keys for external APIs.
 
-By default, this file contains the application's
-`secret_key_base`, but it could also be used to store other credentials such as access keys for external APIs.
+To edit the credentials file, run `bin/rails credentials:edit`. If either the credentials file or the master key does not exist, `bin/rails credentials:edit` will create them.
 
 The secrets kept in credentials file are accessible via `Rails.application.credentials`.
 For example, with the following decrypted `config/credentials.yml.enc`:
 
 ```yaml
-secret_key_base: 3b7cd727ee24e8444053437c36cc66c3
+secret_key_base: 3b7cd72...
 some_api_key: SOMEKEY
 ```
 
-`Rails.application.credentials.some_api_key` returns `SOMEKEY` in any environment.
+`Rails.application.credentials.some_api_key` returns `"SOMEKEY"`.
 
-If you want an exception to be raised when some key is blank, use the bang
+If you want an exception to be raised when some key is blank, you can use the bang
 version:
 
 ```ruby
-Rails.application.credentials.some_api_key! # => raises KeyError: :some_api_key is blank
+# raise KeyError when some_api_key is blank
```

Comment:
The existing comment is the inspect output for the error, rather than prose. That format is used throughout the codebase ([here](https://github.com/rails/rails/blob/95af87f0a4b6274a43853ed68b17e17c21f804f3/activesupport/lib/active_support/core_ext/object/try.rb#L114), [here](https://github.com/rails/rails/blob/95af87f0a4b6274a43853ed68b17e17c21f804f3/activesupport/lib/active_support/core_ext/string/inflections.rb#L70), [here](https://github.com/rails/rails/blob/95af87f0a4b6274a43853ed68b17e17c21f804f3/activerecord/lib/active_record/attribute_methods.rb#L348-L349), etc.) so my preference is to keep using it here too.

Reply:
I wanted to provide some additional context to distinguish this example from the one just above it.  *Por que no los dos?*